### PR TITLE
feat: new blog post

### PR DESCRIPTION
**Pull request by 0xParticle**
Here again with a new post

### CODE REVIEW

File: public/img/blog-posts/lc-bridges/lc1.png
Comment (turtlemoji) [inline public/img/blog-posts/lc-bridges/lc1.png:1]: use https://squoosh.app/ to optimize the image on size and quality

Recommended settings:

- JPG 75% or PNG 75% (only if it has transparency)
- Reduce pallete on, 
- Size: 1920x1080 max

File: public/img/blog-posts/lc-bridges/lc2.png
Comment (turtlemoji) [inline public/img/blog-posts/lc-bridges/lc2.png:1]: idem

File: public/img/blog-posts/lc-bridges/lc4.png
Comment (turtlemoji) [inline public/img/blog-posts/lc-bridges/lc4.png:1]: idem

File: public/img/blog-posts/lc-bridges/lc5.png
Comment (turtlemoji) [inline public/img/blog-posts/lc-bridges/lc5.png:1]: idem

**Review by turtlemoji — CHANGES_REQUESTED** [review]
we should optimize more the images

**Comment by 0xParticle** [comment]
optimized the images using Squoosh

File: src/data/blog.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "id": "lc-bridges",
+    "public": true,
+    "name": "Can we trust interoperability to Light Clients?",
+    "description": "A deep dive into Light Client based bridges",
+    "date": "03/06/24",
+    "tags": ["Consensus", "Verification", "Cross-chain"],
+    "image": "/img/blog-posts/lc-bridges/lc5.png"
```
Comment (0xArdy) [inline src/data/blog.json:9]:
```suggestion
    "image": "/img/blog-posts/lc-bridges/lc5.jpg"
```

**Review by 0xArdy — CHANGES_REQUESTED** [review]

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**